README.md

```diff
@@ -168,6 +170,43 @@ python3 ecosify-strings.py Extensions
 python3 ecosify-strings.py Shared
 ```
 
-## Release 🚀
+## :rocket: Release
 
 Follow the instructions from our [confluence page](https://ecosia.atlassian.net/wiki/spaces/MOB/pages/2460680288/How+to+release)
+
+## Howto update the release notes (semi-automated)
+
+Make sure that `fastlane` and `transifex`-cli is installed.
+
+### Add source release notes to transifex (English)
+
+- Make sure that an _inflight_ version exists in AppStore Connect. If not, create one.
+- Add English text to release notes in AppStore Connect
+- Download metadata from AppStore
+
+    ```bash
+    bundle exec fastlane deliver download_metadata
+    ```
+
+- Push source to Transifex (It is important to limit the push to release_notes.txt)
+
+    ```bash
+    tx push ecosia-ios-search-app.release_notestxt
+    ```
+
+- Wait for translators :hourglass_flowing_sand:
+
+### Add language translations
+
+- Make sure that all languages are translated in the transifex [web interface](https://app.transifex.com/ecosia/ecosia-ios-search-app/release_notestxt/)
```

Comment:
Do we think that we play safe here by mentioning _"all languages"_ ? Questioning in case, in fact, what we really want to have translated are only the languages we are utilizing for the App Store release notes. Raising perhaps to over-clarify